src/swap/sideshift.js

```diff
@@ -112,20 +108,21 @@ async function getAddress(
     : addressInfo.publicAddress
 }
 
-async function checkReplyForError(
-  uri: string,
-  reply: EdgeFetchResponse
-): Promise<any> {
+async function checkReplyForError(reply: EdgeFetchResponse): Promise<any> {
+  let replyJson
   try {
-    await reply.json()
+    replyJson = await reply.json()
```

Comment:
If you want to avoid the `let` binding (but still `catch` the error), then there's a trick!

```
try {
  return (await reply.json())
} catch (e) {
  throw new Error(`SideShift.ai returned error code ${reply.status}`)
}
```